webpack.config.babel.js

```diff
@@ -72,23 +72,28 @@ const baseStyleLoaders = (modules, withSourceMap) => [
 
 
 
-const baseStyleRules = ({disableExtractToFile=false, withSourceMap=true} = {}) =>
-  [{
-    rule: 'exclude',
-    modules: true
-  },
-  {
-    rule: 'include',
-    modules: false
-  }].map(({rule, modules})=> ({
+const baseStyleRules = ({
+  disableExtractToFile = false,
+  withSourceMap = true
+} = {}) =>
+  [
+    {
+      rule: 'exclude',
+      modules: true
+    },
+    {
+      rule: 'include',
+      modules: false
+    }
+  ].map(({ rule, modules }) => ({
     test: /\.(less|css)$/,
     [rule]: [`${__dirname}/node_modules`],
-    use:
-     [
-       disableExtractToFile || !PRODUCTION_BUILD ?
-         'style-loader' : MiniCssExtractPlugin.loader,
-       ...baseStyleLoaders(modules, withSourceMap)
-     ]
+    use: [
+      disableExtractToFile || !PRODUCTION_BUILD
+        ? 'style-loader'
```

Comment:
Minor. I understand this has been automatically prettified, but everywhere else in the code we use the following style:
```
conditionMet ?
  doSomthing :
  doSomethingElse
```

We need to agree on a stricter style guide

Reply:
Will update. But yeah, the `.prettierrc` file I added is a basic one & it should follow whatever is in our `.eslintrc` for most linting but until we agree on that for now there definitely are instances (like this one) where the default conflicts with what we have.

Reply:
I personally prefer the style we're currently using actually.